data/download_data_scripts/README.txt

```diff
@@ -0,0 +1,4 @@
+# Note on downloading data
+
+There are many ways to download/generate data. 
+Besides the scripts on this folder, similar data can de generated using the notebooks to load data on `example_notebooks`.
```

Comment:
Do you think we need to include changes to README into this pull request? @masha-astro  created another one specifically for README file. I just want to make sure that there will not be a conflict.

Reply:
This is a different `README`. It is on the folder `data/download_data_scripts` so there will be no conflicts here. It is just extra information for people using that specific folder

Reply:
Ah, sorry.
I'm good then.